charts/gardener/controlplane/values.yaml

```diff
@@ -455,10 +455,14 @@ global:
         retryPeriod: 2s
         resourceLock: leases
       logLevel: info
-      server:
+      healthServer:
```

Comment:
That's a breaking change. Can you add a release note accordingly?

Reply:
We can make the chart backwards compatible, the gardenlet will automatically choose the "best" configuration. The chart could simply output both the old and new values. We would only move the BC break further in the future, though, when `server` is removed.

Reply:
Na, let's rather make a breaking change right now, instead of having a confusing relationship between values and config.